.github/workflows/lint-app.yml

```
@@ -39,18 +39,19 @@ jobs:
-          yarn eslint $ESLINT_PACKAGES
-
-      - name: Check formatting
-        run: |
-          cd app
-          yarn format:check
-
-      - name: Check multimodal package
-        run: |
-          cd app
-          yarn check
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)